web/ui/static/js/graph/index.js

```diff
@@ -803,6 +817,7 @@ Prometheus.Graph.prototype.handleConsoleResponse = function(data, textStatus) {
       tBody.append("<tr><td colspan='2'><i>no data</i></td></tr>");
       return;
     }
+    data.result = self.limitMetrics(data.result);
```

Comment:
Could we change the wording from "metrics" to "series" here, since "metrics" often in most Prometheus contexts refers to a metric name or all series belonging to a given metric name. Same below in the error message.